lib/Minz/Log.php

```diff
@@ -70,12 +73,31 @@ public static function record ($information, $level, $file_name = null) {
 			     . ' [' . $level_label . ']'
 			     . ' --- ' . $information . "\n";
 
+			self::checkForLogfileSize($file_name);
+
 			if (file_put_contents($file_name, $log, FILE_APPEND | LOCK_EX) === false) {
 				throw new Minz_PermissionDeniedException($file_name, Minz_Exception::ERROR);
 			}
 		}
 	}
 
+	/**
+	 * Make sure we do not waste a huge amount of disk space with old log messages.
+	 *
+	 * This method can be called multiple times for one script execution, but its result will not change unless
+	 * you call clearstatcache() in between. We won't due do that for performance reasons.
+	 *
+	 * @param $file_name
+	 * @throws Minz_PermissionDeniedException
+	 */
+	protected static function checkForLogfileSize($file_name) {
```

Comment:
No objections here, but why not just `checkLogSize` or something? What's the "for" about? :-P

Reply:
Hahaha, lets see if I can manage to add at least one PR in the future without a wording issue from your side ;-)

Reply:
I don't think the PR where I noticed that I somehow typod my own name really counts. :-P